### PR TITLE
CM-APP-44, CM-APP-42 | Change registration screen sequence | Autogenerate CM id

### PR DESCRIPTION
**Pull request by dheeraj-p**

### CODE REVIEW

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt [hunk 1]
```diff
@@ -26,52 +28,65 @@ import android.widget.AdapterView
...
-    AccountCreationClickHandler, CredentialsInputListener, ResponseCallback, AdapterView.OnItemSelectedListener {
-
-    private lateinit var binding: FragmentCreateAccountBinding
 
-    private val disposables = CompositeDisposable()
+class CreateAccountFragment : BaseFragment(){
 
-    private val viewModel: UserAccountsViewModel by sharedViewModel()
+    //private val disposables = CompositeDisposable()
+    //    private var isCriteriaMatch: Boolean = true
```
Comment (AnusreeSajeevan) [inline user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt]: Remove commented code

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt [hunk 2]
```diff
@@ -26,52 +28,65 @@ import android.widget.AdapterView
...
+            android.R.layout.simple_spinner_dropdown_item, android.R.id.text1, viewModel.getYearsToPopulate())
+        binding.spinnerYob.adapter = arrayAdapter
+        arrayAdapter.notifyDataSetChanged()
+        binding.spinnerYob.setSelection(0)
+    }
+    private fun initBindings() {
+        binding.viewModel = viewModel
+//        binding.passwordInputType = getPasswordInputType()
+//        binding.userNameChangeWatcher = UsernameChangeWatcher(this)
+//        binding.passwordChangeWatcher = PasswordChangeWatcher(this)
```
Comment (AnusreeSajeevan) [inline user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt]: Remove commented code

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt (hunk 1 above)
Comment (dheeraj-p) [reply user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt]: done

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt (hunk 2 above)
Comment (dheeraj-p) [reply user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/CreateAccountFragment.kt]: done

**Review by KrupaDesai22 — APPROVED** [review]

**Review by AnusreeSajeevan — APPROVED** [review]
Looks good

**Pull request merged**